style.css

```diff
@@ -80,7 +83,7 @@ button:hover {
   cursor: pointer;
 }
 
-section h1 {
+.sectionTitle {
```

Comment:
Looks like we're using this as a modifier style. I think we can polish up how we approach our `h1`'s.